src/utils.js

```diff
@@ -39,6 +39,33 @@ exports.parseKey = function( key, options ) {
 }
 
 /**
+ * Parses a postgres version string and returns it as an array, e.g.
+ * 
+ * PostgreSQL 9.5.4 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 4.8.2 20140120 (Red Hat 4.8.2-16), 64-bit
+ * 
+ * extracts the version number : 9.5.4 and returns [9, 5, 4]
+ * 
+ * @param   {String} versionString postgres version string
+ * 
+ * @private
+ * @returns {void}
+ */
+function _parsePgVersion( versionString ) {
```

Comment:
don't need _ on functions to state they are private (they are never accessible outside scope)

Reply:
Agreed.